core/src/main/scala/chisel3/internal/package.scala

```diff
@@ -186,9 +181,7 @@ package object internal {
     * import (preferably `import chisel3._`).
     */
   abstract class LegacyBlackBox(
-    params: Map[String, Param] = Map.empty[String, Param]
```

Comment:
seems like this whole class should go away, since there is no `import Chisel._ anymore`. I can open another issue on this one